juniper_official/interface/check-ifl-state-in-out-traffic.rule

```diff
@@ -0,0 +1,425 @@
+/*
+ * Monitors sub interface state and input/output traffic,it notifies when anomalies are found.
+ * Six inputs control detection
+ *
+ *   1) ifd-name, is a regular expression that matches the interfaces
+ *      that you would like to monitor.  By default it's '.*', which matches
+ *      all interfaces. Use something like 'ge.*' to match only gigabit
+ *      ethernet interfaces. 
+ *
+ *   2) ifl-no, is a regular expression that matches the IFL index number that
+ *      you would like to monitor.  By default it's '.*', which matches all
+ *      interfaces. Use something like '0-10*' to match only 0 to 10 IFLs.
+ *
+ *   3) in-stats-name, is a regular expression that matches the sub interface KPI
```

Comment:
This variable not required, we need not give flexibility in jcloud

Reply:
Removed variable.